torch/csrc/jit/passes/quantization.cpp

```diff
@@ -694,6 +694,29 @@ bool isWeightOfConvOrLinear(Value* v) {
       v,
       AtenFuncArgs({{"conv2d", 1}, {"linear", 1}}),
       CallFuncArgs({{"linear", 2}}));
+
+  return result;
+}
+
+bool checkValueIsWeight(Module& module, Graph* graph, Value* v) {
+  if (isWeightOfConvOrLinear(v)) {
+    return true;
+  }
+  bool result = false;
+  auto* self = graph->inputs()[0]; 
```

Comment:
this can be `v->owningGraph()->inputs()[0]` then you don't need to pass around `graph`